build.gradle

```diff
@@ -13,10 +13,10 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.1'
+        classpath 'com.android.tools.build:gradle:3.3.0'
         classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.0'
         classpath 'com.github.dcendents:android-maven-gradle-plugin:2.0'
-        classpath 'com.google.gms:google-services:4.0.1'
```

Comment:
Can you comment on this change? Is this necessary for push? Do we have to update docs and portal for the google plugin version?

Reply:
It's ok to test latest versions, just wanted to make sure it was not a requirement. If you already tested push we can keep it.